src/util/manager/interaction.ts

```diff
@@ -148,6 +153,7 @@ export default class InteractionManager extends Manager {
       // Bulk update all permissions
       await guildCommandManager.permissions.set({ fullPermissions });
     } catch (error) {
+      console.log(error)
```

Comment:
TODO: figure out why handleErr doesnt log anything on startup (or is it only sending to discord)